_includes/manuals/1.18/3.5.2_configuration_options.md

```diff
@@ -578,9 +578,9 @@ Default: The private key file used by puppet
 Time in minutes installation tokens should be valid for, 0 to disable token generation.
 Default: 360 (6 hours)
 
-##### trusted_puppetmaster_hosts
+##### trusted_hosts
```

Comment:
IMHO this is a badly named configuration option. It's very wide and could be interpreted as "fully trust this host". IMHO it should have been `trusted_configuration_management_hosts` or similar. Guess it's too late now.

Reply:
Welcome to open an issue and rename it in core, but for now the docs should match the code :)

Reply:
Hence my ACK but with comments :)